src/_controller/authentification/auth.service.ts

```diff
@@ -4,28 +4,41 @@ import { User } from '../_database/_entity/user/user.entity'
 import { ConfigService } from '@nestjs/config'
 import * as bcrypt from 'bcrypt'
 import { MailService } from 'src/_helper/mail/mail.service'
+import { JwtService } from '@nestjs/jwt';
 
 @Injectable()
 export class AuthService {
     constructor(
         private readonly _userService: UserService,
+        private readonly _jwtService: JwtService,
         private readonly _configService: ConfigService,
         private readonly _mailService: MailService
     ) { }
 
-    async signIn(login: string, password: string): Promise<any> {
-        const userLogin = await this._userService.findLogin(login)
-        console.log(userLogin)
+    async validateUser(login: string, password: string): Promise<any> {
+        if (login.trim() === '' || password.trim() === '') {
```

Comment:
```suggestion
        if (login.trim().length === 0 || password.trim().length === 0) {
```

Je pense que c'est mieux de comparer des int plutôt que des string vide :p

Reply:
Pourquoiiiiiiiiiii ?

Reply:
Je sais plus exactement pourquoi mais dans ma tête c'est plus opti xD Après c'est aussi une question de préférence je pense :D

Reply:
Mouais, je vois pas trop de différence ?

Reply:
"Moauis", la non nonchalance quoi, on dirait moi au stage SNCF

Reply:
Bah non mais avec un "Je sais plus exactement pourquoi mais dans ma tête" a quel moment ça m'aide a savoir ce qui est mieux surtout quand après t'as "question de préférence" ? 😅

Reply:
En fait, comparer une chaîne de caractères c'est comparer chaque caractère un par un, une comparaison de caractère grossomodo c'est pareil qu'une comparaison de nombre, vu que c'est stocké sous cette forme.

Dans le cas où tu as plusieurs caractères, aucun doute que comparer `'1234' == '1234'` est beaucoup moins opti que `1234 = 1234`.

Dans ton cas, vu que la chaîne est vide, je suppose qu'il se base au final sur la length, parce qu'il ne peut pas comparer les caractères. 

Pour la partie "préférence", ce que tu veux check ici, c'est "c'est une chaîne vide", pour faire ça, j'ai plutôt tendance à me dire "ça taille est à 0 --> elle est vide", plutôt que "elle est égal à `''`", mais en soit, ça revient au même ;)

Reply:
> "Mouais", la non nonchalance quoi, on dirait moi au stage SNCF

La non nonchalance c'est l'inverse de la nonchalance ? xD

Reply:
J'ai corrigé le commentaire

Reply:
Ok je comprends mieux ! Merciiiii

Reply:
> J'ai corrigé le commentaire

Je commence à repérer un pattern 🤣